apps/arena/priv/config.json

```diff
@@ -276,8 +276,8 @@
     {
       "name": "antimatter",
       "cooldown_ms": 2000,
-      "execution_duration_ms": 300,
-      "activation_delay_ms": 0,
+      "execution_duration_ms": 800,
+      "activation_delay_ms": 400,
```

Comment:
I tried here with 500 and the orbe shoot feels more like the result of the animation instead of having a weird offset